Chocolatey.Cake.Recipe/Content/notifcations.cake

```diff
@@ -120,83 +120,101 @@ public void SendMessageToTwitter(string message)
 BuildParameters.Tasks.SendNotificationsTask = Task("Send-Notifications")
     .Does(() => 
 {
+    bool dryRun = Context.Argument("dry-run", false);
+
     if (FileExists("./.notifications/discord.txt"))
     {
+        var formattedMessage = System.IO.File.ReadAllText("./.notifications/discord.txt");
+        var messageArguments = BuildParameters.DiscordMessageArguments(BuildParameters.Version);
+
         if (BuildParameters.CanPostToDiscord && BuildParameters.ShouldPostToDiscord)
         {
-            var formattedMessage = System.IO.File.ReadAllText("./.notifications/discord.txt");
-            var messageArguments = BuildParameters.DiscordMessageArguments(BuildParameters.Version);
-            
-            //Information(formattedMessage, messageArguments);
             SendMessageToDiscord(string.Format(formattedMessage, messageArguments));
         }
+        else if (dryRun)
```

Comment:
Shouldn't this if statement be the first one?

If we have requested it to run as `Try-Run`, we shouldn't even try to submit it to the provider.

Reply:
Fair point.  I will get that updated.